File: datasets/IR-Plag/convertai-original-java-rest-python/case-02/plagiarized/L2/04/Kasus2.py
import math

def hitung_area(r):
    return r * r * math.pi

def hitung_volume(area, l):
    return area * l

if __name__ == "__main__":
    r, l = map(float, input("Enter the radius and length of a cylinder: ").split())
    area = hitung_area(r)
    volume = hitung_volume(area, l)
    print(f"The area is {area}")
    print(f"The volume of the cylinder is {volume}")

File: datasets/IR-Plag/convertai-original-java-rest-typescript/case-02/plagiarized/L2/04/Kasus2.ts
import * as readline from 'readline';

/**
 * 
 * @author C166AB8FBBBE55174E16A67530426A8A 19011F43894F06E2BA0CA74061427469
 */
function hitungArea(r: number): number {
    return r * r * Math.PI;
}

function hitungVolume(area: number, l: number): number {
    return area * l;
}

function main(): void {
    const rl = readline.createInterface({
        input: process.stdin,
        output: process.stdout
    });

    rl.question("Enter the radius and length of a cylinder: ", (input) => {
        const [r, l] = input.split(" ").map(Number);
        const area = hitungArea(r);
        const volume = hitungVolume(area, l);
        console.log("The area is " + area);
        console.log("The volume of the cylinder is " + volume);
        rl.close();
    });
}

main();

